go/client/cmd_log_send.go

```diff
@@ -226,19 +233,21 @@ func (c *CmdLogSend) GetUsage() libkb.Usage {
 }
 
 func (c *CmdLogSend) logFiles(status *fstatus) logs {
+	logDir := c.G().Env.GetLogDir()
 	if status != nil {
 		return logs{
 			desktop: status.Desktop.Log,
 			kbfs:    status.KBFS.Log,
 			service: status.Service.Log,
+			start:   filepath.Join(logDir, libkb.StartLogFileName),
```

Comment:
I'd prefer to make this consistent and add this path to `fstatus`.  Then it will be displayed in `keybase status` json (and with one line of code can also be displayed to terminal).